src/components/CodeAndPreview.tsx

```diff
@@ -112,7 +88,7 @@ export function CodeAndPreview({ snippet }: Props) {
         snippet_id: snippet.snippet_id,
         code: code,
         dts: dts,
-        compiled_js: compiledJs,
+        // compiled_js: compiledJs,
```

Comment:
@imrishabh18 note this is one of the biggest changes, we are no longer sending compiled_js to the server. HOWEVER, this was only needed for legacy tscircuit runners, there is now a server bundler/caching system that should eliminate the need for clients (web and cli) to compile JS or compile typescript (the dts)

Still we should be mindful that snippets saved with this will be slightly different